### PR TITLE
airbyte-ci format: make most inputs optional in run-airbyte-ci

### PR DESCRIPTION
**Pull request by alafanechere**
## What
Continue to make `format` automatically run on fork but clean up the work introduced in https://github.com/airbytehq/airbyte/pull/37356 to use the consistent approach in installing and running airbyte-ci via the `run-airbyte-ci` re-usable action.

## How
* Remove the specific "format on fork" job of the `format_check.yml` workflow
* Create a `format` job in the `community_ci.yml` workflow. Make it run automatically in a new `commmunity-ci-auto` environmnet.
* Make most `run-airbyte-ci` inputs optional to avoid requiring secret passing in format.
* Clean up / fix typos in existing workflows

### CODE REVIEW

**Comment by alafanechere** [comment]
* **#37683** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/37683?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alafanechere and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: .github/actions/run-airbyte-ci/action.yml [hunk 1]
```diff
@@ -93,15 +93,14 @@ runs:
     - name: Get start timestamp
       id: get-start-timestamp
       shell: bash
-      run: echo "name=start-timestamp=$(date +%s)" >> $GITHUB_OUTPUT
```
[the remark this answers is not in the document]
Comment (natikgadzhi) [reply .github/actions/run-airbyte-ci/action.yml:96]: Lol, how did this even work then?

File: .github/actions/run-airbyte-ci/action.yml [hunk 2]
```diff
@@ -133,47 +132,46 @@ runs:
...
         S3_BUILD_CACHE_ACCESS_KEY_ID: ${{ inputs.s3_build_cache_access_key_id }}
         S3_BUILD_CACHE_SECRET_KEY: ${{ inputs.s3_build_cache_secret_key }}
         SENTRY_DSN: ${{ inputs.sentry_dsn }}
-        SENTRY_ENVIRONMENT: ${{ steps.determine-install-mode.outputs.install-mode }}
         SLACK_WEBHOOK: ${{ inputs.slack_webhook_url }}
         SPEC_CACHE_BUCKET_NAME: ${{ inputs.spec_cache_bucket_name }}
         SPEC_CACHE_GCS_CREDENTIALS: ${{ inputs.spec_cache_gcs_credentials }}
-    # give the Dagger Engine more time to push cache data to Dagger Cloud
+      run: |
+        airbyte-ci --disable-update-check --disable-dagger-run --is-ci --gha-workflow-run-id=${{ github.run_id }} ${{ inputs.subcommand }} ${{ inputs.options }}
```
Comment (natikgadzhi) [inline .github/actions/run-airbyte-ci/action.yml:144]: TIL there's `disable-dagger-run`!

File: .github/actions/run-airbyte-ci/action.yml [hunk 3]
```diff
@@ -133,47 +132,46 @@ runs:
...
 
-    - name: Collect docker logs on failure
-      id: collect-docker-logs
+    - name: Collect dagger engine logs
+      id: collect-dagger-engine-logs
       if: always()
       uses: jwalton/gh-docker-logs@v2
       with:
-        dest: "./docker_logs"
+        dest: "./dagger_engine_logs"
```
Comment (natikgadzhi) [inline .github/actions/run-airbyte-ci/action.yml:161]: This is essentially just a rename of the output file, right? Still same / similar logs?

File: .github/workflows/community_ci.yml [hunk 4]
```diff
@@ -21,6 +21,43 @@ on:
       - "master"
 
 jobs:
+  format_check:
+    # IMPORTANT: This name must match the require check name on the branch protection settings
+    name: "Check for formatting errors"
+    if: github.event.pull_request.head.repo.fork == true
+    environment: community-ci-auto
```
Comment (natikgadzhi) [inline .github/workflows/community_ci.yml:28]: Does this make it "gated" by a human button press? Do we care?

**Review by natikgadzhi — APPROVED** [review]
I _think_ this is already good — and I agree with @aaronsteers, perhaps we should just keep one implementation of `format_check` that runs on both our PRs and PRs from forks? If there is nothing priviledged, no extra env is needed?

**Pull request merged**